think_big_2023/src/main.py

```diff
@@ -107,46 +107,45 @@ async def image(ctx, prompt: str):
             txt = f'`/image` {prompt}'
             await ctx.reply(file=file, content=txt)
 
+            
 ##################################################
 # Sentiment Analysis
 
 @bot.hybrid_command()
 @discord.ext.commands.guild_only()
-async def sentiment_analysis(ctx, prompt: str):
-    """
-    Given text, the model will return a polarity (positive, negative,
-    neutral) or a sentiment (happiness, anger).
-
-    task: https://huggingface.co/tasks/text-classification
-    model: distilbert-base-uncased-finetuned-sst-2-english
-    size: 268 MB
-    dataset: https://huggingface.co/datasets/sst2
-    source: https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english
-    """
+async def sentiment(ctx, prompt: str):
+    """Predict the polarity (positive, negative, neutral) or a sentiment (happiness, anger)."""
```

Comment:
@oglinuk I was getting errors that these descriptions were too long, and also, nothing after a newline was showing in the UI, so, I shortened these comments.

Reply:
Always good to keep things simple and clear!